core/vmpu/inc/vmpu_kinetis_mem.h

```diff
@@ -31,7 +31,24 @@
  *    | 4 | usb        | NA         | -     |
  *    | 5 | core       | NA         | -     |
  */
-#define UVISOR_TACL_BACKGROUND 0x000007D0U
+
+
+ #define MPU_BUS_MASTER_CORE 0
+ #define MPU_BUS_MASTER_DEBUGGER 1
+ #define MPU_RGDn_WORD2_Mx_UM_NONE_SM_RW     0x00000010U
+ #define MPU_RGDn_WORD2_Mx_UM_NONE_SM_NONE   0x00000018U
+ #define MPU_RGDn_WORD2_Mx_UM_RWX_SM_RWX     0x00000007U
+ #define MPU_RGDn_WORD2_Mx(bus_master, perm) (perm << (bus_master * 6))
+
+#define UVISOR_TACL_DEBUGGER_BACKGROUND ( \
+    MPU_RGDn_WORD2_Mx(MPU_BUS_MASTER_CORE, MPU_RGDn_WORD2_Mx_UM_NONE_SM_NONE) | \
+    MPU_RGDn_WORD2_Mx(MPU_BUS_MASTER_DEBUGGER, MPU_RGDn_WORD2_Mx_UM_RWX_SM_RWX) \
```

Comment:
debugger access it allowed by background 0 region - please remove

Reply:
fixed